packages/mobile/src/analytics/ValoraAnalytics.test.ts

```diff
@@ -21,6 +22,10 @@ jest.mock('react-native-permissions', () => ({}))
 jest.mock('@sentry/react-native', () => ({ init: jest.fn() }))
 jest.mock('src/redux/store', () => ({ store: { getState: jest.fn() } }))
 
+const mockDeviceId = 'abc-def-123'
+const expectedSessionId = 'b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+jest.spyOn(DeviceInfo, 'getUniqueId').mockReturnValue(mockDeviceId)
```

Comment:
Looks like we should apply this change globally in https://github.com/valora-inc/wallet/blob/main/packages/mobile/__mocks__/react-native-device-info.ts